src/edu/washington/escience/myria/operator/agg/AggregatorFactory.java

```diff
@@ -30,4 +31,11 @@
    */
   @Nonnull
   Aggregator get(Schema inputSchema) throws DbException;
+  /**
+   * @param inputSchema
+   * @param pyFuncReg
+   * @return
+   * @throws DbException
+   */
+  Aggregator get(Schema inputSchema, PythonFunctionRegistrar pyFuncReg) throws DbException;
```

Comment:
I don't think you need this new method since `pyFuncReg` is a per-worker argument and doesn't need to be passed here. We can get it through the getter in `Operator` (see my other comments)